src/modify.js

```diff
@@ -66,12 +75,26 @@ function rewriteAllFields(schema, configCallback, context) {
   });
 }
 
+function reorderFields(schema, configOrder) {
+  if (!configOrder) return null;
+
+  const originalOrder = schema['x-jsf-order'];
+  const orderConfig = typeof configOrder === 'function' ? configOrder(originalOrder) : configOrder;
+  const remaining = difference(originalOrder, orderConfig.fields);
```

Comment:
This is also a good place to put an `onWarn` call to flag to devs that a field that was previously ordered was not passed in the new order config

```suggestion
  const nonOrderedFields = difference(originalOrder, orderConfig.fields);
```

Reply:
Yes, let's first align the warn/error API in the other PR https://github.com/remoteoss/json-schema-form/pull/80#discussion_r1684195117